### PR TITLE
[ADD] osi_analytic_segments_stock

### PR DESCRIPTION
**Pull request by max3903**
* Adds the 2 analytic segments in stock moves and scrap

### CODE REVIEW

File: osi_analytic_segments_stock/README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Create or select a scrap order
+* Set the 2 segments and validate
+
+Credits
+=======
+
+Contributors
+------------
+
+* Open Source Integrators <https://www.opensourceintegrators.com>
```
Comment (mgosai) [inline osi_analytic_segments_stock/README.rst:30]: Shouldn't this go under Credits?

**Review by mgosai — APPROVED** [review]

File: osi_analytic_segments_stock/README.rst (hunk 1 above)
Comment (max3903) [reply osi_analytic_segments_stock/README.rst:30]: Not if you want to have the URL...

**Review by patrickrwilson — APPROVED** [review]

File: osi_analytic_segments_stock/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2019 Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "OSI Analytic Segments Stock",
+    "version": "12.0.1.0.1",
```
Comment (bodedra) [inline osi_analytic_segments_stock/__manifest__.py]:
```suggestion
    "version": "12.0.1.0.0",
```

**Review by bodedra — APPROVED** [review]
Code review only. LGTM :+1:

**Pull request merged**